tests/pos-macros/quoted-sym-newtype/Macro_1.scala

```diff
@@ -0,0 +1,10 @@
+import scala.quoted.*
+    
+inline def testMacro = ${ testImpl }
+
+def testImpl(using Quotes): Expr[Unit] = { 
+    import quotes.reflect.*
+    val sym = Symbol.newType(Symbol.spliceOwner, "mytype", Flags.EmptyFlags, TypeRepr.of[String], Symbol.noSymbol)
```

Comment:
In the compiler, the `info` of a type symbol can either be a ClassInfo (if we're defining a class) or a TypeBounds (if we're defining a type parameter, a type alias, an abstract type member or an opaque type). So `String` is not a valid info, it should be `TypeAlias(String)`, but giving the user higher-level methods like `newTypeAlias` and `newTypeBounds` would be better.